### DISCUSSION TITLE
Freezed Package not generating files during build process

### DISCUSSION THREAD

**Discussion by mahmmoudali** [codemagic-ci-cd/codemagic-docs#2554] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1- use 'Freezed' Package in any project and try build using Codemagic CI/CD

### Expected results

Project should build without issues

### Actual results

*Note: I attached the 
1- result of build(bug): file supposed to be generated not found
<img width="873" alt="Screenshot 2024-05-27 at 01 33 38" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/50519599/24af0641-8f24-42b7-bf67-ed220cff289c">

2- prescript command needed to make sure that 'freezed' package working properly
<img width="851" alt="Screenshot 2024-05-27 at 02 07 41" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/50519599/cf665d7c-1ed7-468c-a122-1b3c66ef8f4a">

3- the result of script after commands ran successfully
<img width="851" alt="Screenshot 2024-05-27 at 02 15 16" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/50519599/57945008-2307-4560-a821-ed4d75ae516b">

### Build id (optional)

664c7a3f232bb753ac543322

**Comment by himesh-cm** [comment]
Hey @mahmmoudali, Your latest build was a success. It looks like you managed to resolve this. Please let me know if you are still looking for assistance.

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
